xml/installation-installation-configure-deployer-repos.xml

```diff
@@ -0,0 +1,499 @@
+  %entities;
+]>
+<!-- Converted by suse-upgrade version 1.1 -->
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha.depl.repo_conf_lcm">
+ <info>
+  <title>Software Repository Setup</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer>fs</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>no</dm:translation>
+   <dm:languages/>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    Nodes in &cloud; are automatically installed from the &lcm; server. For this to happen,
+    software repositories containing products, extensions, and the respective
+    updates for all software need to be available on or accessible from the
+    &admserv;. In this configuration step, these repositories are made
+    available. There are two types of repositories:
+   </para>
+   <para>
+    <emphasis role="bold">Product Media Repositories</emphasis>: Product media
+    repositories are copies of the installation media. They need to be
+    directly copied to the &lcm; server, <quote>loop-mounted</quote> from an iso
+    image, or mounted from a remote server via NFS. Affected are &cloudos; and
+    &productname; &productnumber;. These are static repositories; they do not change or receive updates. See <xref linkend="sec.depl.adm_conf.repos.product"/> for setup
+    instructions.
+   </para>
+   <para>
+    <emphasis role="bold">Update and Pool Repositories</emphasis>: Update and
+    Pool repositories are provided by the &scc;. They contain all updates and
+    patches for the products and extensions. To make them available for
+    &cloud; they need to be mirrored from the &scc;. Since their content is
+    regularly updated, they must be kept in synchronization with &scc;. For
+    these purposes, &suse; provides either the &smtool; (&smt;) or the
+    &susemgr;.
+   </para>
+  </abstract>
+ </info>
+ <sect1 xml:id="sec.depl.adm_conf.repos.product">
+  <title>Copying the Product Media Repositories</title>
+  <para>
+   The files in the product repositories for &sls; and &cloud; do not
+   change, therefore they do not need to be synchronized with a remote
+   source. If you have installed the product media from a downloaded ISO image,
+   the product repositories will automatically be made available to the client
+   nodes and these steps can be skipped. Similarly, if you have configured the
+   &lcm; node as an SMT mirror, the product repositories will automatically be
+   made available to the client nodes and these steps can be skipped.
+   Otherwise, it is sufficient to either copy the data (from a remote host or
+   the installation media), to mount the product repository from a remote
+   server via <literal>NFS</literal>, or to loop mount a copy of the
+   installation images.
+   </para>
+   <para>
+    The following product media must be copied to the specified
```

Comment:
I suggest to replace `must be copied to` with `need to reside in` or `need to be located in` or something similar. As the location also need to match in case the repositories are loopback- or nfs-mounted.

Reply:
Done